hapi/hedera-protobufs/services/basic_types.proto

```diff
@@ -1264,6 +1264,11 @@ enum HederaFunctionality {
      * Submit a node signature as part of the Threshold Signature Scheme (TSS) processing.
      */
     TssShareSignature = 98;
+
+    /**
+     * Submit a signature of a state snapshot gossiped to other nodes
+     */
+    StateSignatureTransaction = 99;
```

Comment:
I think this needs a high field number because it will be deprecated before too long and we want to reserve the compact field numbers of fields that will be around a long time.

Reply:
Field numbers less than 16 are compact (single byte), after that it doesn't matter much for quite a range (up to around 2K).